src/main/java/org/javacord/bot/Main.java

```diff
@@ -17,19 +16,42 @@
 import org.javacord.bot.commands.SetupCommand;
 import org.javacord.bot.commands.WikiCommand;
 
-public class Main {
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
 
-    private static Logger logger = LogManager.getLogger(Main.class);
+public class Main {
 
     /**
-     * The entrance point of the bot.
+     * The entry point of the bot.
      *
-     * @param args The first argument should be the bot's token. Every other argument gets ignored.
+     * @param args The bot requires exactly one argument, either a file with the token as content or the token directly.
+     *             If the argument is a relative file path, it is relative to the working directory.
      */
-    public static void main(String[] args) {
+    public static void main(String[] args) throws IOException {
+        Thread.setDefaultUncaughtExceptionHandler(ExceptionLogger.getUncaughtExceptionHandler());
+
+        if (args.length != 1) {
+            System.err.println("This bot requires exactly one argument, " +
+                    "either a file with the token as content or the token directly.\n" +
```

Comment:
`+` operators on new line

Reply:
Yep already fixed.
But good I forgot it, that reminded me that I didn't activate the checkstyle report parsing for the bot builds. :-)